packages/coding-standards/src/CsFixer/ForbiddenPrivateVisibilityFixer.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\CodingStandards\CsFixer;
+
+use PhpCsFixer\Fixer\ConfigurableFixerInterface;
+use PhpCsFixer\Fixer\DefinedFixerInterface;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\FixerDefinition\FixerDefinitionInterface;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+use SplFileInfo;
+
+final class ForbiddenPrivateVisibilityFixer implements DefinedFixerInterface, ConfigurableFixerInterface
+{
+    private const OPTION_ANALYZED_NAMESPACE = 'analyzed_namespaces';
+
+    private $analyzedNamespaces = [];
+
+    /**
+     * {@inheritdoc}
+     */
+    public function configure(array $configuration = null): void
+    {
+        if (array_key_exists(self::OPTION_ANALYZED_NAMESPACE, $configuration) && is_array($configuration[self::OPTION_ANALYZED_NAMESPACE])) {
```

Comment:
This opens up to misconfiguration, if the array key exists it should always be an array, I would rather throw an exception is such a case.